Utilities/CMake/Android/libHttpClient/CMakeLists.txt

```diff
@@ -45,11 +35,19 @@ set(COMMON_SOURCE_FILES
     )
 
 set(ANDROID_SOURCE_FILES
```

Comment:
Just for my info, why did we have to add those now? If it because we no longer have Common source files, etc?

Reply:
So we still have the common source files, but we no longer have the Android-specific source files. So I moved the list of Android-specific files into this file, and added them directly here.